Ukoly/DU_4/Caesar.py

```diff
@@ -0,0 +1,52 @@
+# vytvoření seznamu čísel, která jsou kódy pro písmena abecedy
+rank = list(range(65, 123))
+czech = ['ě', 'š', 'č', 'ř', 'ž', 'ý', 'á', 'í', 'é', 'ň', 'ď', 'ů', 'ú']
+for i in range(91, 97):
+    rank.remove(i)
+
+key = (input('Key: '))
+while key.isnumeric() is False or (int(key) < 0):
+    print('Key has to be positive number.')
+    key = (input('Key: '))
+key = int(key) % 26
+
+plaintext = input('Plaintext: ')
+
+# vytvoření seznamu, kam se budou přidávat zakázané znaky
+forbiden = list()
+for letter in plaintext:
+    if letter.lower() in czech:
+        forbiden.append(letter)
+
+# ošetření zadání stringu s českými znaky:
+while len(forbiden) > 0:
+        print('Please do not use czech alphabet')
+        plaintext = input('Plaintext: ')
+        forbiden.clear()
+
+# vytvoření seznamu, kam se budou přidávat konvertovaná písmena
+convert = list()
+# přidání čísel, které se konvertovaly z písmen, do seznamu
+for letter in plaintext:
+    convert.append(ord(letter))
+
+# vytvoření seznamu pro čísla s klíčem
+convert_key = list()
+# vyloučení ze šifrování jiné znaky než písmena:
+for number in convert:
+    if number in rank:
```

Comment:
Nešlo by kontrolovať, len či to je v nejakom rozmedzí daná hodnota namiesto toho aby si generovala celé pole?
Hint: Áno šlo, pozri si rozpoloženie znakov v ASCII tabuľke